core/ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/part/explorer/project/ProjectExplorerPresenter.java

```diff
@@ -358,6 +358,10 @@ public void onProjectDeleted(DeleteProjectEvent event) {
         }
 
         view.removeNode(toDelete, true);
+
+        if (!view.getRootNodes().isEmpty()) {
+            select(view.getRootNodes().get(0), true);
```

Comment:
Pass `false` as second argument. We don't need to save selection on ones changing.